### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by ytsarev**
Signed-off-by: Yury Tsarev <yury.tsarev@absa.africa>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -254,5 +254,20 @@ To enable verbose debug output declare `DEMO_DEBUG=1` like
 make demo DEMO_URL=https://failover.test.exampledns.tk DEMO_DEBUG=1
 ```
 
+## Release process
+
+* Bump the version in `Chart.yaml`, see [example PR](https://github.com/k8gb-io/k8gb/pull/521).
+* Merge the Pull Request after the review approval
+* Create release tag(replace `v0.8.1` with an actual version)
```
Comment (somaritane) [inline CONTRIBUTING.md:261]: Small fix suggestion
```suggestion
* Create release tag (replace `v0.8.1` with an actual version)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -254,5 +254,20 @@ To enable verbose debug output declare `DEMO_DEBUG=1` like
 make demo DEMO_URL=https://failover.test.exampledns.tk DEMO_DEBUG=1
 ```
 
+## Release process
```
Comment (somaritane) [inline CONTRIBUTING.md:257]: @ytsarev, you might want to update the TOC at the beginning of the document.

Comment (ytsarev) [reply CONTRIBUTING.md:259]: @somaritane updated, thanks for the catch.

**Review by somaritane — CHANGES_REQUESTED** [review]
@ytsarev looks good, just one small fix suggestion left

**Review by kuritka — DISMISSED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (ytsarev) [reply CONTRIBUTING.md]: @somaritane thanks, amended

**Review by somaritane — APPROVED** [review]
@ytsarev LGTM!

**Pull request merged**